ride-service/src/main/kotlin/com/dulfinne/taxi/rideservice/dto/request/LocationRequest.kt

```diff
@@ -1,12 +1,13 @@
 package com.dulfinne.taxi.rideservice.dto.request
 
+import com.dulfinne.taxi.rideservice.util.ValidationKeys
 import jakarta.validation.constraints.NotNull
 
 data class LocationRequest (
 
-    @field:NotNull(message = "Start position can't be empty")
+    @field:NotNull(message = ValidationKeys.START_POSITION_REQUIRED)
     val startPosition: PointRequest,
```

Comment:
When property is marked as not null in Kotlin (without `?`), then NotNull annotation will not work at all